tests/integrations/aws_lambda/test_aws.py

```diff
@@ -22,20 +22,23 @@
 import json
 from sentry_sdk.transport import HttpTransport
 
+FLUSH_EVENT = True
+
 class TestTransport(HttpTransport):
     def _send_event(self, event):
         # Delay event output like this to test proper shutdown
         # Note that AWS Lambda truncates the log output to 4kb, so you better
         # pray that your events are smaller than that or else tests start
         # failing.
-        time.sleep(1)
+        if FLUSH_EVENT:
+            time.sleep(1)
```

Comment:
Hard to understand for maintenance why `FLUSH_EVENT` toggles calling `time.sleep(1)`.

I'd think it would control calling `sentry.flush()` 😕 

Might be a matter of naming?